src/nifs/protogalaxy/accumulator.rs

```diff
@@ -3,18 +3,24 @@ use std::iter;
 use crate::{
     ff::Field,
     halo2curves::CurveAffine,
-    plonk::{RelaxedPlonkTrace, RelaxedPlonkTraceArgs},
+    plonk::{RelaxedPlonkInstance, RelaxedPlonkTrace, RelaxedPlonkTraceArgs},
     poseidon::{AbsorbInRO, ROTrait},
     util,
 };
 
-/// TODO#266 Docs
+/// Represents an accumulator for folding multiple instances into a single instance,
+/// following the accumulation schemes.
 pub struct Accumulator<C: CurveAffine> {
-    /// TODO#266 Docs
-    pub(super) betas: Box<[C::ScalarExt]>,
-    /// TODO#266 Docs
+    /// `φ`: Represents the combined state of all instances & witnesses. It is a summary that
+    /// captures the essential data and relationships from the instances being merged.
     pub(super) trace: RelaxedPlonkTrace<C>,
```

Comment:
I think we can use`PlonkTrace` instead because we don't really need `E_commitment` which is from vanillaFS.

Reply:
https://github.com/snarkify/sirius/pull/331

Reply:
In next PR